tests/acceptance/test_create_organization.py

```diff
@@ -8,7 +10,17 @@ def setUp(self):
         super(CreateOrganizationTest, self).setUp()
         self.user = self.create_user('foo@example.com')
         self.login_as(self.user)
+        settings.PRIVACY_URL = 'https://sentry.io/privacy/'
+        settings.TERMS_URL = 'https://sentry.io/terms/'
```

Comment:
These could be set in src/sentry/utils/pytest/sentry.py we have a bunch if other setup code there and it would avoid mutating global state in a test case.

Reply:
Nice- moved it over. Imo this is the actual case we want to test for. But it might be more appropriate to have these in getsentry instead. Thoughts?